libc/src/sched/sched_getscheduler.h

```diff
@@ -10,7 +10,7 @@
 #define LLVM_LIBC_SRC_SCHED_SCHED_GETSCHEDULER_H
 
 #include "src/__support/macros/config.h"
-#include <sched.h>
+#include "hdr/types/cpu_set_t.h"
```

Comment:
Not used, needs pid_t.h

Reply:
Thanks for the patch! I'm stopping my review here, because this issue pertains to the rest of the PR below I assume.